$OpenBSD: patch-configure_ac,v 1.4 2018/12/21 11:52:05 fcambus Exp $

Don't build a redundant copy of the documentation.
Don't run fc-cache on install.

Index: configure.ac
--- configure.ac.orig
+++ configure.ac
@@ -1260,12 +1260,7 @@ fi
 AC_CHECK_PROGS(MAKEINFO, makeinfo, no)
 
 AC_CHECK_PROGS(TEXI2DVI, texi2dvi, no)
-AC_CHECK_PROGS(PDFTEX, pdftex, no)
-if test x"$TEXI2DVI" = "xno" -o x"$PDFTEX" = "xno"; then
-  AM_CONDITIONAL(BUILD_PDF, false)
-else
-  AM_CONDITIONAL(BUILD_PDF, true)
-fi
+AM_CONDITIONAL(BUILD_PDF, false)
 
 AC_CHECK_PROGS(DOS2UNIX, dos2unix, dos2unix)
 
@@ -3012,12 +3007,7 @@ if test x"$enable_lame" != "xno" ; then
   fi
 fi
 
-AC_PATH_PROGS(FCCACHE, fc-cache, false)
-if test x"$FCCACHE" = "xfalse"; then
-  AM_CONDITIONAL(HAVE_FC_CACHE, false)
-else
-  AM_CONDITIONAL(HAVE_FC_CACHE, true)
-fi
+AM_CONDITIONAL(HAVE_FC_CACHE, false)
 
 if test x"$is_amigaos4" = "xyes"; then
   dnl for AmigaOS4 native resid needs to be configured without dependency tracking
